# Exercício de Casa 🏠 

## Teste de Hipótese

Coletar a altura média de cada uma das alunas, excluindo as alturas da
 Professora, Facilitaroa e Monitora.

**Exercício: Teste z para Altura Média da Turma**

**Cenário:**

Queremos avaliar se a altura média da turma on35 é significativamente diferente da altura média nacional da população feminina brasileira, que é de 160 cm. 

**Objetivo:**

    Verificar se a altura média da turma é significativamente diferente de 160 cm.

**Dados da Amostra:**

    Altura da turma (em cm): vamos criar juntas na aula...


**Perguntas:**


Calcule a estatística *z* e o valor *p* para verificar se a altura média da turma é significativamente diferente de 160 cm.

**Interprete os resultados:** Preciso saber se rejeitou ou não a hipótese nula.

H0 = A altura média das alunas da turma é diferente da média nacional da população feminina brasileira 
H1 = A altura média das alunas da turma é igual a da média nacional da população feminina brasileira 

In [1]:
import numpy as np
from statsmodels.stats.weightstats import ztest

# Dados da amostra
altura_turma = np.array([165,157,163,170,167,161,160,170,157,160,157,168,162,170,167,165,166,158,174,169,163,174,162,170,155,154,158,160,166])
media_nacional = 160

# Teste z para uma amostra
z_statistic, p_value = ztest(altura_turma, value=media_nacional)
print("Teste z para uma amostra:")
print(f"z-statistic: {z_statistic:.4f}, p-value: {p_value:.4f}")

Teste z para uma amostra:
z-statistic: 3.5819, p-value: 0.0003


z-statistic = 3.5819, ou seja, desvio padrão bem alto na média da altura das alunas comparadas a média nacional 
p-value = 0,0003, ou seja, valor muito abaixo de 5%, não rejeitamos H0 pois a altura das alunas realmente é diferente da média nacional. 

**Conclusão**
A altura média da amostra de alturas da turma é significativamente diferente de 160 cm, pois o p-value da amostra foi menor que 5% e por isso não rejeitamos a hipotese nula. Em outras palavras, há evidências estat´titicas significativas para afirmar que a altura média da turma nacional de 160 cm. 